### PR TITLE
Added links to the firmware downloads for Hex products

### PR DESCRIPTION
**Pull request by AndersonRayner**
Links to the firmware for Hex products in case MP doesn't automatically update the firmware and needs to be done manually.

### CODE REVIEW

File: common/source/docs/common-mp-slcan.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ UAVCAN nodes connected.
 .. image:: ../../../images/can-slcan-mpc.png
 
 If the node has bootloader only installed, then firmware will need to be
-uploaded. ``MAINTENANCE`` will be displayed. Click on update firmware. Firmware can be found `here <https://firmware.ardupilot.org/AP_Periph/>`__ and downloaded for the node.
+uploaded. ``MAINTENANCE`` will be displayed. Click on update firmware. AP_Pheriph firmware can be found `here <https://firmware.ardupilot.org/AP_Periph/>`__ and downloaded for the node.  UAVCAN firmware for Hex products (Here GPS, HereFlow) can be downloaded from `here <https://firmware.cubepilot.org/UAVCAN/>`__.
```
Comment (rmackay9) [inline common/source/docs/common-mp-slcan.rst]: Small typo in that it should be "AP_Periph" I think

Comment (AndersonRayner) [reply common/source/docs/common-mp-slcan.rst]: Good catch, fixed

**Comment by Hwurzburg** [comment]
@rmackay9 Randy, do you want me to fix the issues we discussed in Wiki call last week or are you still going to do it?

**Comment by rmackay9** [comment]
@Hwurzburg, I'll try and do it tomorrow, thanks for the reminder!

File: common/source/docs/common-mp-slcan.rst (hunk 1 above)
Comment (TunaLobster) [reply common/source/docs/common-mp-slcan.rst]: Does the Here support CAN? Pretty sure it is just Here2.

**Comment by Hwurzburg** [comment]
I am closing this since the link provided is no longer active

**Pull request closed without merging**